src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -217,11 +217,15 @@ def enable_sequential_cpu_offload(self, gpu_id=0):
 
         device = torch.device(f"cuda:{gpu_id}")
 
+        if self.device.type != "cpu":
+            self.to("cpu", silence_dtype_warnings=True)
+            torch.cuda.empty_cache()    # otherwise we don't see the memory savings (but they probably exist)
```

Comment:
Not very happy about this, didn't see memory savings otherwise

Reply:
Totally fine for me!